syntax = "proto3";
option csharp_namespace = "HSR.MotionCapture.Net.Protos";

enum PacketCode
{
    reserved 2, 3;

    NONE = 0;

    QUIT_NOTIFY = 1;

    HEART_BEAT_REQ = 4;
    HEART_BEAT_RSP = 5;

    FACE_DATA = 6;
    POSE_DATA = 7;
    HAND_DATA = 8;
}